iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensatorModel.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface ShuntCompensatorModel {
+
+    /**
+     * Get the maximum number of sections.
+     */
+    int getMaximumSectionCount();
+
+    /**
+     * Get the susceptance in S of the section with a given section number if it exists.
+     * Throw an exception if such a section does not exist.
+     */
+    double getB(int sectionNum);
+
+    /**
+     * Get the conductance in S of the section with a given section number if it exists.
+     * Throw an exception if such a section does not exist.
+     */
+    double getG(int sectionNum);
+}
```

Comment:
I went to the website documentation about shunt compensators to better understand your changes, but it was not enough for me to understand... I guess you have edited that page of the website in a branch to update that doc for the next release: maybe we should discuss it.

Reply:
I am keeping your comment for now since I have not yet submitted a documentation PR as the APIs are still in construction. I will get back to you after the APIs are fixed.

Reply:
I have created a PR for documentation (and assigned you as reviewer). Could you please take a look at it to see if it is clearer now?

Reply:
@MioRtia I just approved that PR, it looks much clearer to me with your changes.